### PR TITLE
Feat/limit switches

### PR DESCRIPTION
**Pull request by sylent3524**

### CODE REVIEW

File: commands/Claw/MoveClawCommand.py [hunk 1]
```diff
@@ -37,16 +37,22 @@ def getRequirements(self) -> typing.Set[Subsystem]:
...
                 self.armSubsystem.arm.armMotor.set(armSpeed)
                 self.armSubsystem.arm.updateLastAngle()
             else:
                 self.armSubsystem.setHoldArmPosition()
             clawSpeed = self.controller.getClawRotation() * ArmConstants.Claw.speedScale
-            lowerClawLimit = 90 - self.armSubsystem.arm.getAngle()
-            if abs(clawSpeed) > 0 and (clawSpeed > 0 or self.armSubsystem.claw.getAngle() > lowerClawLimit):
+            lowerClawLimit = (
+                90 - self.armSubsystem.arm.getAngle()
+            )  # set to 100 degrees - angle after calibrating claw
```
Comment (mallocassi) [inline commands/Claw/MoveClawCommand.py:52]: Comment and code do not match - let's verify which value is the correct one

**Review by mallocassi — COMMENTED** [review]
Single question on the angle we use to compute lower claw limit

File: commands/Claw/MoveClawCommand.py (hunk 1 above)
Comment (sylent3524) [reply commands/Claw/MoveClawCommand.py:52]: This has to do with the way the claw's calibration. It's slightly off by 10 degrees in reality.
If it is calibrated, this needs to be changed.

**Review by mallocassi — APPROVED** [review]

**Comment by mallocassi** [comment]
Closing in favor of https://github.com/sonic-howl/FRC2023/pull/19

**Pull request closed without merging**